unattended_installer/install_functions/checks.sh

```diff
@@ -175,6 +175,26 @@ function checks_arguments() {
 
 }
 
+function check_dist() {
```

Comment:
If you use a `case` instead of `if`s, do you think you could have a clearer code?

Reply:
Done [b86f1e1](https://github.com/wazuh/wazuh-packages/pull/1317/commits/b86f1e10985a72a3acdd2306edd49f25aa80da7b)

Reply:
Better with `if`s. Please revert.